crates/sui-cluster-test/src/test_case/coin_merge_split_test.rs

```diff
@@ -27,7 +27,9 @@ impl TestCaseImpl for CoinMergeSplitTest {
         let gas_obj = sui_objs.swap_remove(0);
 
         let signer = ctx.get_wallet_address();
-        let primary_coin = sui_objs.swap_remove(0);
+        let mut sui_objs_2 = ctx.get_sui_from_faucet(Some(1)).await;
```

Comment:
Can you explain about the fix?

Reply:
Oh basically faucet only gives one coin object, and previously the tests are expecting 5 objects from faucet.